docs/spec/openapi.yml

```diff
@@ -1635,6 +1674,38 @@ components:
         password:
           type: string
           example: root
+    DatabaseClusterComponents:
+      type: array
+      description: components related data
+      items:
+        type: object
+        x-go-type-name: DatabaseClusterComponent
+        properties:
+          status:
+            type: string
+          name:
+            type: string
+          type:
+            type: string
+          age:
+            type: string
+          restarts:
+            type: integer
+          ready:
+            type: string
+          containers:
+            type: array
+            items:
+              $ref: '#/components/schemas/DatabaseClusterComponentContainer'
+    DatabaseClusterComponentContainer:
+      type: object
+      properties:
+        name:
+          type: string
+        age:
+          type: string
+        restarts:
+          type: integer
```

Comment:
On the UI design there are also those "Ready" and "Status" columns, are they going to be figured out by the FE somehow from the existing data?

Reply:
They're part of the response in https://github.com/percona/everest/blob/1838b0e702f0d55fd7b9b0173803134930283586/docs/spec/openapi.yml#L1677